media-sound/supercollider/supercollider-3.8.0.ebuild

```diff
@@ -0,0 +1,106 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit cmake-utils readme.gentoo-r1
+
+DESCRIPTION="An environment and a programming language for real time audio synthesis."
+HOMEPAGE="http://supercollider.github.io/"
+SRC_URI="https://github.com/supercollider/supercollider/releases/download/Version-${PV}/SuperCollider-${PV}-Source-linux.tar.bz2"
+
+RESTRICT="mirror"
+
+LICENSE="GPL-2 gpl3? ( GPL-3 )"
+SLOT="0"
+KEYWORDS="~x86 ~amd64"
+
+IUSE="
+	avahi debug emacs +fftw gedit +gpl3 ide jack portaudio qt5 server +sndfile
+	static-libs vim wiimote
+	cpu_flags_x86_sse cpu_flags_x86_sse2"
+
+REQUIRED_USE="
+	ide? ( qt5 )
```

Comment:
Is there good reason to have this? As far as I can see, USE=ide does not add another dependency, so why not drop that flag and conflict and let `SC_IDE` be controlled by qt5 as well?